### PR TITLE
Output original prm

### PR DESCRIPTION
**Pull request by tjhei**
ASPECT now outputs a file named original.prm in the output directory with the exact content of the parameter it got started with.

### CODE REVIEW

**Comment by tjhei** [comment]
I think this came up in a discussion with @gassmoeller and we agreed this was very useful to have.

**Comment by jdannberg** [comment]
There where several occasions where I wished I'd have had this feature, thank you for implementing this!

**Comment by tjhei** [comment]
updated.

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -3174,6 +3174,7 @@ \subsection{Overview}
 total 932
 -rw-rw-r-- 1 bangerth bangerth  11134 Dec 11 10:08 depth_average.gnuplot
 -rw-rw-r-- 1 bangerth bangerth  11294 Dec 11 10:08 log.txt
+-rw-rw-r-- 1 bangerth bangerth     42 Dec 11 10:07 original.prm
```
Comment (gassmoeller) [inline doc/manual/manual.tex]: I wonder how you created this file on Wolfgang's machine in the past? :smirk:

Comment (tjhei) [reply doc/manual/manual.tex]: psst. :wink:

**Pull request merged**